invenio_rdm_records/requests/community_inclusion.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Community addition request."""
+
+from invenio_i18n import lazy_gettext as _
+from invenio_records_resources.services.uow import RecordCommitOp, RecordIndexOp
+from invenio_requests.customizations import RequestType, actions
+
+from ..proxies import current_rdm_records_service as service
+
+
+#
+# Actions
+#
+class SubmitAction(actions.SubmitAction):
+    """Submit action."""
+
+    def execute(self, identity, uow):
+        """Execute the submit action."""
+        record = self.request.topic.resolve()
+        # Set the record's title as the request title.
+        self.request["title"] = record.metadata["title"]
+        super().execute(identity, uow)
+
+
+class AcceptAction(actions.AcceptAction):
+    """Accept action."""
+
+    def execute(self, identity, uow):
+        """Accept record into community."""
+        # Resolve the topic and community - the request type only allow for
+        # community receivers and record topics.
+        record = self.request.topic.resolve()
+        community = self.request.receiver.resolve()
+
+        assert not record.parent.review
```

Comment:
minor: should this be an error raised instead? From parsing through the service code, I believe it won't be possible to reach this state, but if it's because of the ES-based `_exists` check, maybe that's our safety net? In any case, maybe a comment here would help.